Contest - 1 - October 18 2020/cpp contest codes/cycleDetection.cpp

```diff
@@ -0,0 +1,31 @@
+// Complete the has_cycle function below.
+
+/*
+ * For your reference:
+ *
+ * SinglyLinkedListNode {
+ *     int data;
+ *     SinglyLinkedListNode* next;
+ * };
+ *
+ */
+bool has_cycle(SinglyLinkedListNode* head) {
+    SinglyLinkedListNode *t = head;   // here t will point to head node
+SinglyLinkedListNode *r = head;
```

Comment:
Correct the indentation here. Make the code more readable.

Reply:
ok

Reply:
for correcting indentation again i had to make pr ?

Reply:
Naah, I have corrected it. Just make a note of it for future.